src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import file.reader.FileReader;
```

Comment:
Imports should be grouped according to their packages. All imports from the 'file.reader' package should be together, followed by 'file.writer', 'model', 'service.converter', 'service.operation', 'service.report', and 'service.shop'.